.github/scripts/clone_engine_repo.conf

```diff
@@ -0,0 +1 @@
+DEFAULT_BRANCH=
```

Comment:
You need to set this to something sane, otherwise it'll default to whatever the github default branch is (currently 2_X_DEV). Take a look in one of the older 2.x branches, like 2.1, to see what it's set to there.